Modules/Core/Common/include/itkShapedImageNeighborhoodRange.h

```diff
@@ -347,7 +347,7 @@ class ShapedImageNeighborhoodRange final
     /**  Returns a reference to the current pixel. */
     reference operator*() const ITK_NOEXCEPT
     {
-      return reference(m_ImageBufferPointer, m_ImageSize, m_OffsetTable, m_NeighborhoodAccessor, m_Location + *m_CurrentOffset);
+      return reference{m_ImageBufferPointer, m_ImageSize, m_OffsetTable, m_NeighborhoodAccessor, m_Location + *m_CurrentOffset};
```

Comment:
I'm fine with using curly braces instead of parentheses, but it only looks like a STYLE issue to me. (Actually I originally wanted to use curly braces on that line of code but KWStyle complained, at that time. If I remember correctly!) So are you sure that this change really has such an impact on the performance? If so, do you have a clue why?

Reply:
Yes, older versions of KWStyle will complain about `};` :disappointed: , but this has been fixed in newer versions :sunny: .

I did observe the speed up, but I do not know the details of why this makes a difference to the compiler.

Reply:
Glad to hear KWStyle no longer complains about  this ` };` syntax!

With which specific compiler version did you observe the speed-up? Because it sounds like a compiler issue to me! I just tried, but did not observe a significant performance difference between curly braces and parentheses on VS2017 (Version 15.8.9).

Anyway, the code change you're proposing here is perfectly fine to me.